privaterelay/templates/profile.html

```diff
@@ -135,6 +135,8 @@
                       aria-label="{% ftlmsg 'profile-label-edit' %}"
                       placeholder="{% ftlmsg 'profile-label-placeholder' %}"
                       class="relay-email-address-label js-relay-email-address-label ff-Met"
+                      {% comment %} Require at least one non-whitespace character: {% endcomment %}
```

Comment:
As this isn't a [multi-line comment](https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#std:templatetag-comment), I think you can get away with just the [single-line syntax](https://docs.djangoproject.com/en/3.2/topics/templates/#comments): 
```suggestion
                      {# Require at least one non-whitespace character:  #}
```

Reply:
TIL! That's just what I got after pressing <kbd>Ctrl</kbd>+<kbd>/</kbd> :P